src/Chainweb/Pact/Backend/Compaction.hs

```diff
@@ -164,35 +164,64 @@ runCompactM e a = runReaderT (unCompactM a) e
 
 -- | Prepare/Execute a "$VTABLE$"-templated query.
 execM_ :: ()
-  => Text -- ^ "$VTABLE$"-templated query
```

Comment:
Added query logging to all of this; this should become more like chainweb logging, though. That will be in a separate PR